<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleDataSettingEditWidget</class>
 <widget class="QWidget" name="ParticleDataSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="colorGroupBox">
       <property name="title">
        <string>Color</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="valueRadioButton">
          <property name="text">
           <string>Color by value</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="valueComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="customColorRadioButton">
          <property name="text">
           <string>Custom Color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="ColorEditWidget" name="customColorWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="colorMapGroupBox">
     <property name="title">
      <string>Colormap</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidgetContainer" name="colorMapWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1">
     <item>
      <widget class="QLabel" name="pointSizeLabel">
       <property name="text">
        <string>Point Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pointSizeSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="TransparencyWidget" name="transparencyWidget" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWidgetContainer</class>
   <extends>QWidget</extends>
   <header>misc/qwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>valueRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customColorRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>customColorWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorMapGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
